app/pulp/app/serializers/fields.py

```diff
@@ -11,6 +11,14 @@ class ContentRelatedField(DetailRelatedField):
     """
     queryset = models.Content.objects.all()
 
+    def use_pk_only_optimization(self):
```

Comment:
I moved this up into the `DetailRelatedField` so it will work generally and this block can be removed.